spec/system/admin/reports/users_and_enterprises_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'system_helper'
+
+describe "Users & Enterprises reports" do
+  include AuthenticationHelper
+
+  let!(:enterprise) { create(:supplier_enterprise) }
+
+  before do
+    login_as_admin
+    visit main_app.admin_report_path(report_type: 'users_and_enterprises')
+  end
+
+  it "displays the report" do
+    find("[type='submit']").click
```

Comment:
This is brittle and will fail if we ever add any other form to the page. Other specs use `click_button "Go"`. It also looks more like what the user does.

Reply:
Good point, fixed by https://github.com/openfoodfoundation/openfoodnetwork/pull/11041/commits/41dead853d3aa9059098ca1cd4c69dc592db32ea